lms/static/js/spec/student_account/account_settings_factory_spec.js

```diff
@@ -145,6 +157,10 @@ define(['backbone',
                 var accountSettingsView = createAccountSettingsPage();
 
                 AjaxHelpers.respondWithJson(requests, Helpers.createAccountSettingsData());
+                AjaxHelpers.respondWithJson(requests, [{
+                    'time_zone': 'America/Guyana',
+                    'description': 'America/Guyana (ECT, UTC-0500)'
+                }]);
```

Comment:
Instead of hard-coding this data in each test, can you make a helper method (or at least a common variable to reference)? See https://github.com/edx/edx-platform/blob/master/lms/static/js/spec/student_account/helpers.js#L36-L43 as an example.